cmd/repo.go

```diff
@@ -488,13 +493,34 @@ func (index *RepoIndex) buildStacksFromIndex(repoName string, Stacks []Stack) ([
 
 	for id, value := range index.Projects {
 
-		//	Stacks = append(Stacks, Stack{repoName, id, value[0].Version, value[0].Description, "*" + value[0].DefaultTemplate})
-		Stacks = append(Stacks, Stack{repoName, id, value[0].Version, value[0].Description})
-
+		Stacks = append(Stacks, Stack{repoName, id, value[0].Version, value[0].Description, "*" + value[0].DefaultTemplate})
 	}
 	for _, value := range index.Stacks {
-		Stacks = append(Stacks, Stack{repoName, value.ID, value.Version, value.Description})
+		Templates := value.Templates
+
+		templatesString := ""
```

Comment:
might be worth calling this templatesListString to avoid confusion with temaplteString

Reply:
changed